app/server-control/directives/kvm-console.js

```diff
@@ -30,11 +30,18 @@ window.angular && (function(angular) {
                 return false;
               };
 
+              var KVMconn = 0;
               function connected(e) {
                 $log.debug('RFB Connected');
+                KVMconn = 1;
               }
               function disconnected(e) {
                 $log.debug('RFB disconnected');
+                if (KVMconn == 0) {
```

Comment:
Why (KVMconn == 0) ? Shouldn't this be 1? 
Isn't 1 when there is a KVM session active?

Reply:
Expected:
1. The alert will only be displayed on the webpage of users who have not successfully connected to the KVM session.
2. For users who have successfully connected the KVM session, even if they press refresh or F5, this alert will not be displayed.